nestkernel/connection_manager.cpp

```diff
@@ -853,8 +876,7 @@ nest::ConnectionManager::connect_tripartite( NodeCollectionPTR sources,
   // Set flag before calling cb->connect() in case exception is thrown after some connections have been created.
   set_connections_have_changed();
 
-  cb->connect();
-  delete cb;
```

Comment:
why was this removed?

Reply:
Previously, `cb` was a base-class pointer to a specific builder instance created by `get_conn_builder()`. Now `cb` is a local variable holding a `ConnBuilder` instance, which internally manages the specific builders.